scripts/sanity_chk/sanitylib.py

```diff
@@ -3352,10 +3377,11 @@ def __init__(self):
         self.detected = []
         self.connected_hardware = []
 
-    def load_device_from_cmdline(self, serial, platform):
+    def load_device_from_cmdline(self, serial, platform, serial_pty=''):
         device = {
             "serial": serial,
             "platform": platform,
+            "serial_pty": serial_pty,
```

Comment:
shouldnt this be in the yaml schema?

Reply:
Added to https://github.com/zephyrproject-rtos/zephyr/blob/master/scripts/sanity_chk/hwmap-schema.yaml

Reply:
this was actually not needed :) I misread the diff. You depend on --device-serial-pty which means you never parse a hardware map file.

Reply:
Removed from yaml schema.